### ISSUE TITLE
How to finetune a PI oriented SR model from the one you submitted in PIRM2018?

### ISSUE THREAD

**Issue by W-yk** [xinntao/ESRGAN#32]
I try to build a PI oriented SR model based on your excellent work.  Would you please give some guide on how to fine tune your model  or elaborate some training details for PI oriented training ..

**Comment by WenlongZhang0724** [comment]
If you try to build a new perceptual-oriented SR model: First, you need to use MSE loss to train SR model as pre-train of Generator; second, training GAN to obtain a well-trained G and D net.  (refer to the setting of SRGAN and ESRGAN, such as learning rate, etc. ); Third, the well-trained G and D net as pre-train to further finetune G according to your need.
If you just try to finetune ESRGAN (or SRGAN),  you just do the second and third above.

**Comment by WenlongZhang0724** [comment]
> I try to build a PI oriented SR model based on your excellent work. Would you please give some guide on how to fine tune your model or elaborate some training details for PI oriented training ..

If you try to build a new perceptual-oriented SR model: First, you need to use MSE loss to train SR model as pre-train of Generator; second, training GAN to obtain a well-trained G and D net. (refer to the setting of SRGAN and ESRGAN, such as learning rate, etc. ); Third, the well-trained G and D net as pre-train to further finetune G according to your need.
If you just try to finetune ESRGAN (or SRGAN), you just do the second and third above.

**Comment by W-yk** [comment]
Thanks for the reply. In your paper, you mentioned that higher PSNR doesn't guaranteed a better perceptual index. Isn't the current training code focus on psnr?  So how can I tell when the training for PI is enough with psnr as the validation output?..

**Comment by WenlongZhang0724** [comment]
> Thanks for the reply. In your paper, you mentioned that higher PSNR doesn't guaranteed a better perceptual index. Isn't the current training code focus on psnr? So how can I tell when the training for PI is enough with psnr as the validation output?..

In general, the last 100 K (total 500 K ) iteration SR results perform better PI and visual results. You could use PI to test the model in this range. I sent an e-mail to you(18250763559@163.com)
